MapboxNavigation/VisualInstruction.swift

```diff
@@ -20,11 +20,23 @@ extension VisualInstruction {
             mv.backgroundColor = .clear
             mv.scale = UIScreen.main.scale
             mv.visualInstruction = self
-            return mv.imageRepresentation
+            let image = mv.imageRepresentation
+            return shouldFlipImage(side: side) ? image?.withHorizontallyFlippedOrientation() : image
         }
         guard blackAndWhiteManeuverIcons.count == 2 else { return nil }
         return CPImageSet(lightContentImage: blackAndWhiteManeuverIcons[1], darkContentImage: blackAndWhiteManeuverIcons[0])
     }
+    
+    func shouldFlipImage(side: DrivingSide) -> Bool {
+        let leftDirection = [.left, .slightLeft, .sharpLeft].contains(maneuverDirection)
+        
+        switch maneuverType {
+        case _ where maneuverDirection == .uTurn, .takeRoundabout, .turnAtRoundabout, .takeRotary:
```

Comment:
Looks like this is effectively switching on `maneuverDirection`, not `maneuverType`.